pages/public/404.vue

```diff
@@ -17,12 +16,12 @@
 import NavigationBar from "~/components/NavigationBar.vue";
 
 export default {
-    components: {
+  components: {
     NavigationBar
   },
   methods: {
-  gotoBack() {
-      $nuxt._router.go(-1)
+    gotoBack() {
+      $nuxt._router.go(-1);
```

Comment:
Não é a mesma coisa que router.back()? Acho mais verboso

Reply:
não sei man... se for altero agora

Reply:
Cara, coloquei .back() no local e foi hein